packages/catppuccin-vsc/src/theme/tokens/index.ts

```diff
@@ -107,7 +107,7 @@ export default function tokens(context: ThemeContext): TextmateColors {
       },
     },
     {
-      name: "Punctuation",
+      name: "Operators",
       scope: [
         "keyword.operator",
         "punctuation.accessor",
```

Comment:
This says punctuation but `keyword.operator` is not punctuation (`=`, `==`, `!`, etc) and the `punctuation.accessor` scope is like `?` in TypeScript. We could split them off? I'm not sure what the rest here do...